package.json

```diff
@@ -87,13 +87,13 @@
   },
   "dependencies": {
     "@emotion/css": "11.13.5",
-    "@grafana/aws-sdk": "0.5.0",
-    "@grafana/data": "11.4.0",
-    "@grafana/experimental": "2.1.4",
-    "@grafana/prometheus": "11.4.0",
-    "@grafana/runtime": "11.4.0",
-    "@grafana/schema": "11.4.0",
-    "@grafana/ui": "11.4.0",
+    "@grafana/aws-sdk": "0.6.0",
+    "@grafana/data": "11.4.0-205469",
+    "@grafana/experimental": "2.1.6",
+    "@grafana/prometheus": "11.4.0-205469",
+    "@grafana/runtime": "11.4.0-205469",
+    "@grafana/schema": "11.4.0-205469",
+    "@grafana/ui": "11.4.0-205469",
```

Comment:
A new combobox component added in prometheus ui with https://github.com/grafana/grafana/pull/95647
But the way it was imported is breaking the build for this datasource. 
Right before its merge `v11.4.0-205469` was the latest release. So I used it.
I'll fix the import asap.